src/routes/app/(flows)/claim-project/steps/add-ethereum-address/add-ethereum-address.svelte

```diff
@@ -48,24 +48,20 @@
 
         const { forge, username, repoName } = GitProjectService.deconstructUrl($context.gitUrl);
 
-        try {
-          await github.getFundingJson(
-            username,
-            repoName,
-            dripsJsonTemplate(
-              $walletStore.address ?? unreachable(),
-              $walletStore.network.name
-                ? $walletStore.network.name === 'homestead'
-                  ? 'ethereum'
-                  : $walletStore.network.name
-                : unreachable(),
-            ),
-          );
+        await github.getFundingJson(
```

Comment:
What'll happen now on error?
Is it caught somewhere higher on the call stack?

Reply:
Yes the Stepper catches any error and displays it in the same error modal style